includes/class-wc-gateway-ppec-cart-handler.php

```diff
@@ -351,7 +351,7 @@ public function display_mini_paypal_button() {
 		$settings = wc_gateway_ppec()->settings;
 
 		// billing details on checkout page to calculate shipping costs
-		if ( ! isset( $gateways['ppec_paypal'] ) || 'no' === $settings->cart_checkout_enabled || 0 === WC()->cart->get_cart_contents_count() ) {
+		if ( ! isset( $gateways['ppec_paypal'] ) || 'no' === $settings->cart_checkout_enabled || 0 === WC()->cart->get_cart_contents_count() || '0.00' === WC()->cart->get_total( 'edit' ) ) {
```

Comment:
To avoid the possibility of this var type changing in the future, and then failing on this strict string condition, I think we should use `WC()->cart->needs_payment()` instead.

Reply:
Good idea. Pushed a commit with the change.